xunit-autolabeler-v2/ast_parser/core/analyze.py

```diff
@@ -13,39 +13,53 @@
 # limitations under the License.
 
 import json
+import os
 from os import path
-from typing import List, Set, Tuple
+from typing import Any, Dict, List, Set, Tuple
 
-from . import polyglot_drift_data as pdd, polyglot_parser, yaml_utils
+from ast_parser.lib import constants as lib_constants
```

Comment:
Only alias the import if two modules share the same name or if import has an inconveniently long name.

Reply:
There are other `constants` files, so I wanted to be clear on which one this was. (Even if it's the only one we import here.)